### DISCUSSION TITLE
Error while executing JS

### DISCUSSION THREAD

**Discussion by zhengtulymGh** [Budibase/budibase#5357] [Help] [answered]
<img width="1625" alt="v消息20220412-103116" src="https://user-images.githubusercontent.com/13768025/162867534-193f72ad-6c11-4342-b4a3-af7a6c93d9e8.png">

<img width="719" alt="v消息20220412-103116(1)" src="https://user-images.githubusercontent.com/13768025/162867553-a6b53b81-6b38-41f9-9d1e-128d12e74c8b.png">

I would like to format the date value by javascript code.

**Comment by melohagan** [comment, marked as answer]
Hey @zhengtulymGh 

The bindings in JavaScript take a different format. If you click one of the suggestions on the left-hand menu you will see how it should appear.

It'll be along the lines of: `$("New Form.Fields.report_date")`

**Reply by zhengtulymGh** [reply]
Thanks
